src/classes/FORM_ServiceGiftEntry_TEST.cls

```diff
@@ -230,19 +230,88 @@ public with sharing class FORM_ServiceGiftEntry_TEST {
             'New template names should return true from `FORM_ServiceGiftEntry.checkNameUniqueness`');
     }
 
+    @isTest
+    static void shouldRetrieveTemplateWithCRUDErrors() {
+        UTIL_PermissionsMock utilPermissionsMockInstance = new UTIL_PermissionsMock(false, 'CRUD');
+        UTIL_Permissions utilPermissionsMock = (UTIL_Permissions) Test.createStub(UTIL_Permissions.class,                   utilPermissionsMockInstance);
```

Comment:
Throughout this class, add line-breaks for the long lines to improve readability

Reply:
fixed